recipe/meta.yaml

```diff
@@ -1,11 +1,11 @@
-{% set version = "2022.02.2" %}
+{% set version = "2022.02.2+485" %}
 
 package:
   name: rstudio
   version: {{ version }}
```

Comment:
Not sure what happens if we put a "+" in the version; we could replace it with a "." if we are fine with a 4-part version

Reply:
```suggestion
  version: {{ version | replace("+", ".") }}
```